tool/autoqa/lib/bert-canonical-annotator.py

```diff
@@ -46,20 +56,44 @@ def template_query(cat, query_canonical='', prefix='', value='', suffix=''):
             f"show me a {query_canonical} with {prefix} {value} {suffix} .".split(),
             f"{question_start} has {prefix} {value} {suffix} ?".split()
         ]
+    if cat == 'property_true' or cat == 'property_false':
+        return [
+            f"show me a {query_canonical} with {prefix} .".split(),
+            f"{question_start} has {prefix} ?".split()
```

Comment:
I would add a comment here noting that property_false also uses "with" instead of "without" (ie, we're trying to form "show me a restaurant with no parking" as opposed to "show me a restaurant without parking"), cause that tripped me just now.